pubspec.lock

```diff
@@ -1,6 +1,20 @@
 # Generated by pub
 # See https://dart.dev/tools/pub/glossary#lockfile
```

Comment:
I don't think you should check this file in?  Pretty sure this is made when you do a packages get

Reply:
Done, I added this file to the gitignore.